phpseclib/Common/Functions/Objects.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/**
+ * Common Object Functions
+ *
+ * PHP version 5
+ *
+ * @category  Common
+ * @package   Functions\Objects
+ * @author    Jim Wigginton <terrafrost@php.net>
+ * @copyright 2016 Jim Wigginton
+ * @license   http://www.opensource.org/licenses/mit-license.html  MIT License
+ * @link      http://phpseclib.sourceforge.net
+ */
+
+namespace phpseclib\Common\Functions;
+
+/**
+ * Common Object Functions
+ *
+ * @package Functions\Objects
+ * @author  Jim Wigginton <terrafrost@php.net>
+ */
+abstract class Objects
+{
+    /**
+     * Accesses a private variable from an object
+     *
+     * @param Object $obj
+     * @param string $var
+     * @return mixed
+     * @access private
```

Comment:
I've updated the phpdoc lines. Is `@access ...` needed when visibility is declared in the function declaration? I figure the phpdoc blocks are in pretty bad state and at some later date I figure I'll make a PR dedicated to fixing them.